modules/application_load_balancer/main.tf

```diff
@@ -6,16 +6,6 @@ resource "aws_security_group" "tfe_lb_allow" {
   vpc_id = var.network_id
 }
 
-resource "aws_security_group_rule" "tfe_lb_allow_inbound_http" {
-  type              = "ingress"
-  from_port         = 80
-  to_port           = 80
-  protocol          = "tcp"
-  cidr_blocks       = ["0.0.0.0/0"]
-  description       = "Allow HTTP (port 80) traffic inbound to TFE LB"
-  security_group_id = aws_security_group.tfe_lb_allow.id
-}
-
```

Comment:
we can actually keep this! the security concern comes where there is _only_ an HTTP listener, and no HTTPS listener. having an HTTP listener is sane when it redirects to HTTPS (which we do here)